IBAnimatableTests/CornerDesignableTests.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  CornerDesignableTests.swift
+//  IBAnimatable
+//
+//  Created by Steven on 4/24/17.
+//  Copyright © 2017 IBAnimatable. All rights reserved.
+//
+
+import XCTest
+@testable import IBAnimatable
+
+class CornerDesignableTests: XCTestCase {
+
+  // MARK: - testCornerRadius
+
+  func testCornerRadius() {
+    helper_testCornerRadius_UIView(element: AnimatableView())
+    helper_testCornerRadius_UIView(element: AnimatableCheckBox())
+    helper_testCornerRadius_UIView(element: AnimatableImageView())
+    helper_testCornerRadius_UIView(element: AnimatableLabel())
+    helper_testCornerRadius_UIView(element: AnimatableStackView())
+    helper_testCornerRadius_UIView(element: AnimatableTableViewCell())
+    helper_testCornerRadius_UIView(element: AnimatableStackView())
+    helper_testCornerRadius_UIView(element: AnimatableView())
+    helper_testCornerRadius_UICollectionViewCell(element: AnimatableCollectionViewCell())
+  }
+
+  private func helper_testCornerRadius_UIView(element: MockUIViewCornerDesignable) {
```

Comment:
After going back to refactor this now I remember why it is the way that it is.

Basically, there's nothing to test for `cornerSides` that isn't already tested under `cornerRadius` because setting `cornerSides` just calls `configureCornerRadius()` behind the scenes.